src/core/libmaven/mzMassSlicer.cpp

```diff
@@ -237,27 +257,346 @@ void MassSlices::algorithmC(float ppm, float minIntensity, float rtWindow) {
     cerr << "#algorithmC" << slices.size() << endl;
 }
 
-//Function to check if slice is already present in cache
-mzSlice*  MassSlices::sliceExists(float mz, float rt) {
-    pair< multimap<int, mzSlice*>::iterator,  multimap<int, mzSlice*>::iterator > ppp;
-    // putting all mz slices in cache in a particular range in ppp
-    ppp = cache.equal_range( (int) (mz* 10) );
-    multimap<int, mzSlice*>::iterator it2 = ppp.first;
+mzSlice* MassSlices::sliceExists(float mzMinBound,
```

Comment:
So this method is not being used in algorithmB, right?

Reply:
Not anymore, no.